src/reporter/tests/run_tests.sh

```diff
@@ -5,8 +5,10 @@ docker build -t quantumleap ../../../
 docker-compose up -d
 sleep 12
 
-docker run -ti --rm --network tests_reportertests quantumleap pytest reporter/tests
+cd ../../../
+pytest src/reporter/ --cov-report= --cov-config=.coveragerc --cov-append --cov=src/ 
```

Comment:
also, if coverage adds too much overhead for the local day-to-day development, would you be interested in having a simple way to disable coverage for the runs?

Reply:
Yes, this would definitely be a nice to have!